release-notes/1.0/RC2_Contributor_List.md

```diff
@@ -0,0 +1,404 @@
+* [David Kean (1)](https://github.com/dotnet/corefx/commits/master?author=davkean@microsoft.com)
+* [Vijay Kota (1)](https://github.com/dotnet/corefx/commits/master?author=vijaykota@users.noreply.github.com)
+* [Chris Rummel (1)](https://github.com/dotnet/corefx/commits/master?author=crummel@microsoft.com)
+* [Chamila Chulatunga (1)](https://github.com/dotnet/corefx/commits/master?author=chamila.chulatunga@gmail.com)
+* [William Godbe (1)](https://github.com/dotnet/corefx/commits/master?author=wigodbe@microsoft.com)
+* [BrennanConroy (1)](https://github.com/dotnet/corefx/commits/master?author=brecon@microsoft.com)
+* [Zlatko Knezevic (1)](https://github.com/dotnet/corefx/commits/master?author=zlakne@microsoft.com)
+* [Bill Wert (1)](https://github.com/dotnet/corefx/commits/master?author=billwert@microsoft.com)
+* [bjjones (1)](https://github.com/dotnet/corefx/commits/master?author=brandon1.jones@intel.com)
+* [stukselbax (1)](https://github.com/dotnet/corefx/commits/master?author=stukselbax@gmail.com)
+* [Ben Pye (1)](https://github.com/dotnet/corefx/commits/master?author=ben@curlybracket.co.uk)
+* [Bart J.F. De Smet (1)](https://github.com/dotnet/corefx/commits/master?author=bartde@microsoft.com)
+* [jashook (1)](https://github.com/dotnet/corefx/commits/master?author=jashoo@microsoft.com)
+* [Ashley Mannix (1)](https://github.com/dotnet/corefx/commits/master?author=admin@kodr.com.au)
+* [Andrew Schwartzmeyer (1)](https://github.com/dotnet/corefx/commits/master?author=andschwa@microsoft.com)
+* [mihailik (1)](https://github.com/dotnet/corefx/commits/master?author=mihailik@gmail.com)
+* [mirhagk (1)](https://github.com/dotnet/corefx/commits/master?author=mirhagk@gmail.com)
+* [moozzyk (1)](https://github.com/dotnet/corefx/commits/master?author=pawelka@microsoft.com)
+* [nbilling (1)](https://github.com/dotnet/corefx/commits/master?author=nbilling@microsoft.com)
+* [Andrew Arnott (1)](https://github.com/dotnet/corefx/commits/master?author=andarno@microsoft.com)
+* [Noah Falk (1)](https://github.com/dotnet/corefx/commits/master?author=noahfalk@microsoft.com)
+* [Nate McMaster (1)](https://github.com/dotnet/corefx/commits/master?author=nate.mcmaster@microsoft.com)
+* [MyungJoo Ham (1)](https://github.com/dotnet/corefx/commits/master?author=myungjoo.ham@samsung.com)
+* [Mike Rousos (1)](https://github.com/dotnet/corefx/commits/master?author=MikeRou@Microsoft.com)
+* [Miguel de Icaza (1)](https://github.com/dotnet/corefx/commits/master?author=miguel@gnome.org)
+* [Michael Aird (1)](https://github.com/dotnet/corefx/commits/master?author=michael.aird@gmail.com)
+* [Matt Mitchell (1)](https://github.com/dotnet/corefx/commits/master?author=matchell@outlook.com)
+
+## .NET Core Tools
+
+* [Sridhar Periyasamy (195)](https://github.com/dotnet/cli/commits/master?author=sridhper@microsoft.com)
+* [Bryan Thornbury (165)](https://github.com/dotnet/cli/commits/master?author=brthor@microsoft.com)
```

Comment:
I believe rel/1.0.0 has been merged back to master since 3/21, so that would make sense although I'd expect to see some extra commits from the dotnet native folks.

With dates set up to today, the diff is only two commits
https://github.com/dotnet/cli/compare/rel/1.0.0...rel/1.0.0-preview1